Taskfile.yml

```diff
@@ -216,6 +228,21 @@ tasks:
     vars:
       QEMU_RUN_SCRIPT: ./scripts/qemu_run.sh
 
+  run-iso:
+    deps:
+      - task: deps:check-cmd
+        vars: {CMD: "swtpm"}
```

Comment:
Hmm, would it be possible *not* requiring swtpm, or have that dependency as an option?

Reply:
swtpm is already an optional dependency unless for the `run-iso` `run-disk` targets.
This entry does only assure swtpm is present before calling `scripts/qemu_run.sh` :+1:

Reply:
Since swtpm is optional, it will be removed for convenience :+1: